iroh/tests/cmd/start_status_stop.trycmd

```diff
@@ -0,0 +1,14 @@
+```
+$ iroh start gateway
```

Comment:
this has to test individual services because we use a `HashSet` to determine sets, and order isn't deterministic, which makes testing the output nondeterministic ☹️

Reply:
you are looking for a BTreeSet, or just use a vec it’s fine